src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserSessionPool.cs

```diff
@@ -89,11 +85,7 @@ internal void BestEffortCleanup()
                 TdsParserStateObject session = _cache[i];
```

Comment:
GitHub won't let me comment on it, but the comment above says "This is called from a ThreadAbort - ensure that it can be run from a CER catch"... thread aborts don't exist in coreclr.